src/VisualStudio/CSharp/Impl/xlf/CSharpVSResources.cs.xlf

```diff
@@ -674,8 +679,8 @@
       </trans-unit>
       <trans-unit id="Prefer_is_null_over_ReferenceEquals">
         <source>Prefer 'is null' over 'object.ReferenceEquals(..., null)'</source>
-        <target state="translated">Preferovat „is null“ před „object.ReferenceEquals(..., null)“</target>
-        <note />
+        <target state="needs-review-translation">Preferovat „is null“ před „object.ReferenceEquals(..., null)“</target>
```

Comment:
why needs-review? the text didn't change

Reply:
a note was added.